package.json

```diff
@@ -4,10 +4,11 @@
   "description": "Standalone version of the Spherical contours notebook",
   "main": "index.js",
   "scripts": {
-    "data": "node bin/download-data.js",
-    "build": "rollup -c",
+    "clean": "rm -rf ./node_modules ./public ./vendor",
```

Comment:
@Fil Please comment if it's OK for you... It's always somewhat touchy to use automate "rm -rf"

Reply:
I think it's ok